Sources/Database/Database.swift

```diff
@@ -363,15 +363,7 @@ private struct _Client {
   func fetchSubscriptionTeammates(ownerId: Models.User.Id) -> EitherIO<Error, [Models.User]> {
     return self.rows(
       """
-    SELECT "users"."email",
-           "users"."episode_credit_count",
-           "users"."github_user_id",
-           "users"."github_access_token",
-           "users"."id",
-           "users"."is_admin",
-           "users"."name",
-           "users"."subscription_id",
-           "users"."rss_salt"
+    SELECT "users".*
```

Comment:
Database decoding errors are super opaque and we get them whenever we add new fields to the user model so I think `*` is prob the way to go for these queries...